### ISSUE TITLE
High efficiency transform cytosolic 8x dicodon library into RQC-del strains

### ISSUE THREAD

**Issue by kychen37** [rasilab/github_demo#1]
GanttStart: 2022-06-27
GanttDue: 2022-07-08

### Background

I want to see which pathways 8x dicodon motifs follow in terms of stability and degradation. Integrating the library into hel2-del, not5-del, and syh1-del strains will show which motifs follow which regulatory paths and are ultimately degraded compared to my WT library. Hel2 and syh1 may have redundant effects on NGD substrates, not5 should regulate nonoptimal/slow-decoding dicodons. 

### Strategy

Transform pHPHS1142 cytosolic 8x dicodon library into scHP1408 (BY4741-1::hel2-del), scHP1784 (BY4741-1::not5-del), and syh1-del (BY4741-1::syh1-del). Use liquid recovery as in https://github.com/rasilab/rqc_aggregation_aging/issues/98 and then library prep as in https://github.com/rasilab/rqc_aggregation_aging/issues/104, because that experiment (BY4741-1 WT cells with 1142 library) had good sequencing results.

### Experiment Links

https://github.com/rasilab/rqc_aggregation_aging/blob/master/experiments/kchen_exp65_higheff_transfo_8xdicodon_rqc-del_strains.md

### Brief conclusion

Analysis in https://github.com/rasilab/rqc_aggregation_aging/issues/117
Note that the spike-in strains don't currently (2023-10-02) have glycerol stocks! Need to be transformed and stocked, plasmids are in -20C.

#### Checklist before closing issue

- [x] Are all plasmids, oligos, and cell lines in their correct location?
- [ ] Is the plasmid / cell line entered into our lab database?
- [x] Is the plasmid map given appropriate name and moved to the `lab_database` folder on Snapgene?
- [x] Is the plasmid map pushed to https://github.com/rasilab/snapgene_maps/?
- [x] Is the lab notebook link up-to-date without any broken links to images?
- [x] Are there any intermediate plates or reagents that need to be disposed off?

**Comment by kychen37** [comment]
@rasi 

**Update**
- I will transform pHPSC1142 8xdicodon library into scHP1408 (hel2-del) and scHP1784 (not5-del) next week via liquid recovery. If/when syh1-deletion is generated I will also transform into syh1-del.
- I should use spike-ins, I need to check that the ones I have will work for this library because I can't remember

**Comment by rasi** [comment]
@kychen37 This plan sounds good. Just to confirm, the spike-ins will still be wild-type cells. We want to have the same cells with the same plasmids across all experiments for normalization.

**Comment by kychen37** [comment]
@rasi
**Update**
- I am in the middle of troubleshooting/optimizing this transformation for the different strains. I was hoping it would be as straightforward as repeating my WT library transformation exactly as before, but the efficiency I got was super low when I tried it. So now I am working at small scale to optimize the transformation conditions for each strain in order to get something that will scale up reasonably well.

**Comment by kychen37** [comment]
@rasi 

## Update
- I figured out why my efficiency was low and will scale up for the real library transformations next week using hel2-del, not5-del, and recently generated syh1-del

**Comment by rasi** [comment]
@kychen37 What was the reason for low efficiency? 

I noticed that you are doing midipreps. @heungwonpark has switched back to multiple minipreps of scraped cells to get higher yield. This seems to avoid ethanol carryover and quicker overall. Talk to him about this.

Does your *ΔNot5* grow significantly slower?  This would be a good confirmation that you do not have suppressor mutations or an incomplete knockout, since [Veltri 2021](https://www.biorxiv.org/content/10.1101/2021.12.01.470814v1.full) say that the growth phenotype is the reason why they don't find *Not5* in their screen.

> While some of these strains are not present in the deletion strain collection due to their severe growth phenotype (dhh1Δ and not4/5Δ),

**Comment by kychen37** [comment]
@rasi it was ethanol carryover from midipreps. I switched to minipreps after talking Heungwon which should fix the issues.

Hmm I have not noticed our Not5-del (scHP1784) strain growing slower than other strains, I can double check the KO by sangerseq

**Comment by rasi** [comment]
@kychen37 Ok, minipreps should fix it.

Yes, please check the KO by amplifying with two outside primers. I know @heungwonpark had issues with strains like Dcp2 where the Kan cassette will go in, but the gene itself won't be deleted. So it will look like a KO from one side. @heungwonpark can chat more about this.

**Comment by kychen37** [comment]
## Update

@rasi As you suspected our not5-del strain scHP1784 is not really deleted despite having the kanmx cassette, see various colony PCRs below using primer pairs in various regions of not5 in scHP1784 (not5-del) vs WT. 
I will order the new primers for knocking out not5. In the meantime I will keep going forward with syh1-del and hel2-del libraries

<img src="https://user-images.githubusercontent.com/43553196/180102649-9eca0c1d-0420-4b1b-a46c-f6fc63ef91b7.jpg" width=500px>

**Comment by kychen37** [comment]
I noticed that we have primers to check for full-length hel2 in the database but don’t see if this pcr was run in the database entry’s lab notebook. @rasi or @heungwon Do you remember if a full-length pcr was run on the scHP1408 hel2-del strain?

**Comment by rasi** [comment]
@kychen37 Good point. Don't think we checked, but will be useful as a +ve control. You don't even need a full length PCR as much as PCRs across both arms (one primer in the insert and one outside on both sides). I suspect Hel2 is fine for two reasons:

1. Knocking out Hel2 is not thought to have a growth phenotype.
2. Hel2 knockout produces the expected reporter phenotype of increased mRNA and protein levels in Park 2019. So does Asc1, which has both a growth phenotype and a reporter phenotype.

**Comment by kychen37** [comment]
## Update
@rasi 
- I expect to have the representation I need for the hel2-del and syh1-del libraries by Monday
- I plan to start library prepping these two libraries on Monday, and most likely finish up prepping when I get back from DC
- excluding Not5-del for now because it's very slow-growing, though this suggests the KO worked at least. I streaked out a few  colonies to try and obtain single colonies of each, and am now waiting on these single colonies to grow up
